### PR TITLE
update tng.org mappings

### PR DESCRIPTION
**Pull request by SimonGreenhill**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [X] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

Updating a few tng.org mappings.

### CODE REVIEW

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 1]
```diff
@@ -549,7 +549,7 @@ Greenhill-2015-2525-1269	1269	swollen
 Greenhill-2015-2525-1270	1270	to be thirsty	847	THIRSTY
 Greenhill-2015-2525-1271	1271	to be tired	1757	TIRED
 Greenhill-2015-2525-1272	1272	toenail	2662	TOENAIL
-Greenhill-2015-2525-1273	1273	tomorrow morning		
+Greenhill-2015-2525-1273	1273	tomorrow morning	1339	MORNING
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Greenhill-2015-2525.tsv:552]: Is this a specific concept that only refers to the morning of the next day?

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv [hunk 2]
```diff
@@ -793,7 +793,7 @@ Greenhill-2015-2525-1965	1965	to jump	1829	JUMP
 Greenhill-2015-2525-1966	1966	shoot, plant	3358	SPROUT (SHOOT)
 Greenhill-2015-2525-1967	1967	firelight		
 Greenhill-2015-2525-1968	1968	vulva	1837	VULVA
-Greenhill-2015-2525-1969	1969	short-piece-of-wood		
+Greenhill-2015-2525-1969	1969	short-piece-of-wood	3222	TWIG
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Greenhill-2015-2525.tsv:796]: Could it also be a branch? Then 3223 BRANCH OR TWIG would fit better.

**Review by AnnikaTjuka — COMMENTED** [review]
Thanks for updating the mappings, @SimonGreenhill! I just have two clarification questions.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv (hunk 1 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:552]: I think it's just for tomorrow, remove the mapping?

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv (hunk 2 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:796]: I think a branch wouldn't be short? but happy to change if you want

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv (hunk 1 above)
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:552]: Yes, then I'd remove the mapping.

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv (hunk 2 above)
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:796]: True. Let's leave the mapping as is.

Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv:796]: ok

File: concepticondata/conceptlists/Greenhill-2015-2525.tsv (hunk 1 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Greenhill-2015-2525.tsv]: ok, done.

**Comment by SimonGreenhill** [comment]
@AnnikaTjuka -- I've also just unmapped "cracking sound" from "SOUND OR NOISE"

**Review by AnnikaTjuka — APPROVED** [review]
Many thanks, feel free to merge!

**Pull request merged**